src/cgtop/cgtop.c

```diff
@@ -672,7 +673,7 @@ static void help(void) {
                "  -n --iterations=N   Run for N iterations before exiting\n"
                "  -b --batch          Run in batch mode, accepting no input\n"
                "     --depth=DEPTH    Maximum traversal depth (default: %u)\n"
-               "  -M --machine=       Show container\n"
+               "  -M --machine=       Show container (incompatible when cgroup path is specified)\n"
```

Comment:
The code is not prepared for the scenario you are describing. You would need to add a column to to each record describing the root used to determine the values, and I doubt it would really add to the usability of this utility.

So no, I am not convinced this is a good idea, but even if you wanted to enable such a functionality later, you could still add comma separated values to the option or something like that.

Anyway, I don't want to have the final word on this. Let @poettering decide that.